src/Factory/Api/Mastodon/Notification.php

```diff
@@ -70,7 +70,9 @@ public function createFromNotification(Notifications\Entity\Notification $Notifi
 			$status = null;
 		}
 
-		return new MstdnNotification($Notification->id, $type, $Notification->created, $account, $status);
+		$dismissed = $Notification->dismissed;
+
+		return new MstdnNotification($Notification->id, $type, $Notification->created, $account, $status, $dismissed);
```

Comment:
If `$dismissed` isn't used for anything else I'd rather you inline the value like this:
```suggestion
		return new MstdnNotification($Notification->id, $type, $Notification->created, $account, $status, $Notification->dismissed);
```